exporter/datadogexporter/metrics_translator.go

```diff
@@ -261,12 +263,21 @@ func mapMetrics(cfg config.MetricsConfig, prevPts *ttlmap.TTLMap, md pdata.Metri
 			attributeTags = attributes.TagsFromAttributes(rm.Resource().Attributes())
 		}
 
+		host, hostFromAttrs := metadata.HostnameFromAttributes(rm.Resource().Attributes())
+
+		// Report the host as running
+		runningMetric := metrics.DefaultMetrics("metrics", pushTime)
+		if hostFromAttrs {
+			runningMetric[0].Host = &host
+		}
+		series = append(series, runningMetric...)
+
 		ilms := rm.InstrumentationLibraryMetrics()
 		for j := 0; j < ilms.Len(); j++ {
 			ilm := ilms.At(j)
-			metrics := ilm.Metrics()
-			for k := 0; k < metrics.Len(); k++ {
-				md := metrics.At(k)
+			metricsArray := ilm.Metrics()
```

Comment:
Note: these codecov warnings are due to the lack of unit tests for `mapMetrics`, they'll be fixed in another PR